src/messaging/tests/echo/echo_responder.cpp

```diff
@@ -42,16 +43,30 @@ namespace {
 
 // The EchoServer object.
 chip::Protocols::Echo::EchoServer gEchoServer;
+chip::Protocols::UserDirectedCommissioning::UserDirectedCommissioningServer gUDCServer;
```

Comment:
The other pieces of the code use the instance that is provided by the class - is this server being used for a different purpose in the tests?

Reply:
no, that was a straggler (in test code) but fixed, thanks